src/wxdoughnutchart.cpp

```diff
@@ -58,7 +70,7 @@ const wxChartCommonOptions& wxDoughnutChart::GetCommonOptions() const
 void wxDoughnutChart::Initialize(const wxDoughnutChartData &data,
                                  const wxSize &size)
 {
-    const wxVector<wxChartSliceData>& slices = data.GetSlices();
```

Comment:
As noted above we should keep GetSlices() as that is a better name than GetValue().